client/src/Components/Input/Input.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { Story } from '@storybook/react';
+import Input, { InputProps } from '../Input/Input';
+
+// eslint-disable-next-line import/no-anonymous-default-export
+export default {
+  title: 'Input',
+  component: Input,
+  decorators: [(Story: any) => <Story />],
+};
+
+export const PrimaryInput: Story<InputProps> = (basePropsData) => <Input {...basePropsData} />;
+
+PrimaryInput.args = {
```

Comment:
basePropsData

Reply:
.args jest elementem Storybooka, chyba, że nie o to chodzi, nie rozumiem